covid19-dashboard/dc-covid19/src/SideNav.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+export default function SideNav(props: {handleScrollOnRef, date: string}) {
+    return (
```

Comment:
Also this menu generation could be from a config.json file

Reply:
Resolved.